src/utilities/MiscUtilities.ts

```diff
@@ -8,17 +8,17 @@ import {
     Snowflake,
     TextChannel
 } from "discord.js";
-import {CommonRegex} from "../constants/CommonRegex";
-import {DefinedRole} from "../definitions/Types";
-import {IGuildInfo, ISectionInfo} from "../definitions";
-import {MongoManager} from "../managers/MongoManager";
-import {GuildFgrUtilities} from "./fetch-get-request/GuildFgrUtilities";
-import {MessageUtilities} from "./MessageUtilities";
-import {StringBuilder} from "./StringBuilder";
-import {AdvancedCollector} from "./collectors/AdvancedCollector";
-import {ButtonConstants} from "../constants/ButtonConstants";
-import {PermsConstants} from "../constants/PermsConstants";
-import {Logger} from "./Logger";
+import { CommonRegex } from "../constants/CommonRegex";
```

Comment:
I think it's because I switch between two different editors; one editor adds a space around the import while the other one doesn't.

What do you think is the best? Is there a way to enforce this via ESLint?

Reply:
Another change should have been pushed. I personally do not mind what style we end up using, as long as it's consistent.